Description: temporarly disable demo plugin
 this patch should be removed, when a new 
 version appears, where already no binaries
 in demo/plugin folder.
Author: Anton Gladky <gladk@debian.org>
Last-Update: 2015-06-16

Index: gnuplot/demo/Makefile.am.in
===================================================================
--- gnuplot.orig/demo/Makefile.am.in
+++ gnuplot/demo/Makefile.am.in
@@ -9,7 +9,6 @@ epslatex.ps epslatex.tex random.tmp stri
 BINARY_FILES = binary1 binary2 binary3
 
 SUBDIRS = $(PLUGIN_DIR)
-DIST_SUBDIRS = plugin
 
 DEMO = all.dem
 
